sdk/keyvault/azure-keyvault-keys/pyproject.toml

```diff
@@ -1,3 +1,49 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]  # Requires 61.0.0 for dynamic version
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "azure-keyvault-keys"
+authors = [
+    {name = "Microsoft Corporation", email = "azurekeyvault@microsoft.com"}
+    ]
+description = "Microsoft Azure Key Vault Keys Client Library for Python"
+requires-python = ">=3.7"
+license = {text = "MIT License"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python",
```

Comment:
This will not stick around. I am merely including a package that successfully builds with `python -m build` so that I can verify that all of CI still works properly. I will back this change out before merging this PR.